_inc/client/my-plan/checklist-progress-card/style.scss

```diff
@@ -0,0 +1,94 @@
+.checklist__header {
+	display: flex;
+	flex-direction: row;
+
+	&-main {
+		display: flex;
+		flex: 1 1;
+		flex-direction: column;
+		align-items: stretch;
+	}
+
+	&-secondary {
+		display: flex;
+		flex: 2 1;
+		flex-direction: row;
+		align-items: center;
+		justify-content: flex-end;
+	}
+
+	&-progress {
+		display: flex;
+		flex-direction: row;
+		font-size: 14px;
+		white-space: nowrap;
+		margin-bottom: 5px;
+	}
+
+	&-progress-text {
+		display: flex;
+		flex: 1 1;
+		margin: 0;
+		color: $muriel-gray-500;
+	}
+
+	&-progress-number {
+		display: flex;
+		color: $muriel-gray-500;
+		padding-left: 1em;
+	}
+
+	&-summary {
+		font-size: 12px;
+		line-height: 24px;
+		color: $muriel-gray-500;
+		cursor: pointer;
+	}
+
+	&-action {
+		position: absolute;
+		top: 0;
+		right: 0;
+		width: 48px;
+		height: 100%;
+		border-left: 1px solid $muriel-gray-0;
+		cursor: pointer;
+
+		.gridicon {
+			fill: $muriel-gray-300;
+			vertical-align: middle;
+			align-items: center;
+			transition: transform 0.15s cubic-bezier( 0.175, 0.885, 0.32, 1.275 ), color 0.2s ease-in;
+		}
+
+		.checklist.is-expanded & {
+			.gridicon {
+				transform: rotate( 180deg );
+			}
+		}
+
+		.accessible-focus &:focus {
+			box-shadow: inset 0 0 0 2px $muriel-blue-300;
+		}
+	}
+
+	.checklist.is-expanded &-action {
+		.gridicon {
+			transform: rotate( 180deg );
+		}
+	}
+
+	.progress-bar__progress {
+		background-color: $muriel-hot-green-400;
+	}
+
+	.dops-button.is-primary {
+		margin-right: 0;
+		background: $green-primary;
+		border-color: $green-secondary;
```

Comment:
Yes, it's a bit weird to have blue and green CTAs on the same page I think:

![image](https://user-images.githubusercontent.com/426388/57454174-82e93e80-7268-11e9-9c89-64156a4f6765.png)


Maybe that new button should be blue?

Since we are comparing buttons, maybe we should look at not uppercasing the text in that new button?

Reply:
@jeffgolenski ☝️

Reply:
If we have time here, I think it would be prudent to build some logic in. The color is less important. What's important is that we only have one primary CTA show at any given time. 

Now that we're implementing the checklist I'd argue the only primary CTA should be the "continue jetpack setup" button. Let's make it blue for now and make all other CTAs white.

Once the jetpack checklist has been completed, that button should disappear completely. then we can make that "view settings" button the primary CTA.

Reply:
I've opened a simple follow-up, I don't want to get into all of these changes in this PR. Feel free to add details @jeherve and @jeffgolenski 

#12405